person.rb

```diff
@@ -13,6 +16,10 @@ def can_use_services?
     of_age? || @parent_permission ? true : false
   end
 
+  def correct_name
+    puts @name
+  end
```

Comment:
- [ ] The method should return the @name, not print it to the console, kindly make the changes so the method return the @name